lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -47,47 +47,57 @@
     font-size: 90%;
   }
 }
-.exam-timer {
-  line-height: 56px;
-  background-color: #e5eaec;
-  padding-left: 42px;
-  padding-right: 32px;
-  border-left: 4px solid #14a6ea;
-  margin: 0 auto;
-  color: #4e575b;
-  font-size: 14px;
-  a {
-    color: #0979ba;
+.proctored_exam_status {
+  // STATE: Fixed to viewport (used for scrolling)
+  &.is-fixed {
+    @extend %ui-depth4;
+    box-shadow: 0 3px 3px $shadow-d1;
+    position: fixed;
+    top: 0;
+    width: 100%;
   }
-  span.pull-right {
-    color: #646161;
+  .exam-timer {
     line-height: 56px;
-    b {
-      color: #414040;
-    }
-  }
-  &.low-time {
-    color: #cdd7db;
-    background-color: #4F585C;
+    background-color: #e5eaec;
+    padding-left: 42px;
+    padding-right: 32px;
+    border-left: 4px solid #14a6ea;
+    margin: 0 auto;
+    color: #4e575b;
+    font-size: 14px;
     a {
-      color: #fff;
-      text-decoration: underline;
+      color: #0979ba;
     }
     span.pull-right {
-      color: #cdd7db;
+      color: #646161;
+      line-height: 56px;
       b {
+        color: #414040;
+      }
+    }
+    &.low-time {
+      color: #cdd7db;
+      background-color: #4F585C;
+      a {
         color: #fff;
+        text-decoration: underline;
+      }
+      span.pull-right {
+        color: #cdd7db;
+        b {
+          color: #fff;
+        }
       }
     }
+    &.warning {
+      border-left-color: #feb93e;
```

Comment:
Can this color value use a pre-defined one from our ``_variables.scss`` file (https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_variables.scss#L56)?